lib/elix/responders/lists.ex

```diff
@@ -11,7 +11,7 @@ defmodule Elix.Responders.Lists do
   show lists - Displays all lists
   """
   respond ~r/show lists\Z/i, msg do
-    reply(msg, render_items(List.all_names))
+    send(msg, render_items(List.all_names))
```

Comment:
I ended up using a `case`:

```elixir
response =
  case List.all_names do
    [] -> "There are no lists yet."
    lists -> render_items(lists)
  end
```

...but I'd be very interested in null-object-like patterns in Elixir. The only (crazy) thing I can think of offhand is a custom struct that you `defimpl` enumerable on so when I `Enum.map` over the result of `List.all_names` it returns a string. But that feels pretty heavy-handed.